src/mopi/inference.py

```diff
@@ -0,0 +1,51 @@
+from mopi.run_dev import run_dev
+from mopi.type import Experiment
+from mopi.blocks.pipeline import Pipeline
+
+from mopi.type import (
+    Experiment,
+    DatasetSplit,
+)
+from mopi.data.dataloader import PandasDataLoader
+from mopi.type import PreprocessConfig
+import pandas as pd
+from mopi.library.evaluation.classification import classification_metrics
+from mopi.blocks.io import load_pipeline
+from mopi.constants import Const
+
+from typing import Tuple, List
+
+
+def run_inference(pipeline: Pipeline, texts: List[str]) -> Tuple[int, float]:
+
+    text_with_fake_labels = [[text, 0] for text in texts] + [["dummy_text", 1]]
```

Comment:
Our current setup expects that ``test_dataset`` has also ``labels`` so I am adding random labels to it and making sure both categories are available. 

Later a more elegant approach would be to support test_dataset not having labels, but it requires boolean and checks that might mess with the current setup.